e2e_tests/utils.py

```diff
@@ -85,11 +94,13 @@ def image_to_png(*, image: Image) -> PngImageFile:
     return PngImageFile(png_buf)
 
 
+def a_key(name: str):
+    return "".join(random.choices(population=name.replace("-", ""), k=10)).upper()
```

Comment:
why `random.choices`?? it just garbles the received string (slug)

Reply:
Instead, we could generate a random ASCII string. Project key should be as unique as possible and it was a tricky way to achieve this.

Reply:
Done